src/index.ts

```diff
@@ -34,7 +34,7 @@ const localExecuter = LocalExecuter.getExecuter()
  * the communication attempt with the radio.
  */
 export const executeCommunication = (
-  serializedActionObject: string,
+  serializedActionObject: any,
   forwardingAddress?: string
 ): Promise<string> => {
```

Comment:
Since a ActionObject.serialize() is now returning a JSON object, won't this Promise need to be adjusted to return the "any" type?

Also, won't we also have to adjust PAM to both take and return the JSON object before we can merge this code?